src/routes/userRoutes.js

```diff
@@ -6,9 +6,10 @@ import validationHandler from '../validation/validationHandler';
 
 const userRoutes = express.Router();
 
-const { createUser } = UserController;
-const { checkUserExists } = UserMiddlewares;
+const { createUser, verifyEmail } = UserController;
+const { checkUserExists, isUserVerified } = UserMiddlewares;
 
 userRoutes.post('/signup', userValidation, validationHandler, checkUserExists, createUser);
+userRoutes.get('/verify', isUserVerified, verifyEmail);
```

Comment:
I don't know o, but is it not automatically after signUp the user gets the email?

Reply:
its immediately

Reply:
So immediately after creating a user he should get an email?

Reply:
yes